advisories/published/2025/JLSEC-0000-mnsapys1s-17kdzwq.md

```diff
@@ -0,0 +1,24 @@
+```toml
+schema_version = "1.7.3"
+id = "JLSEC-0000-mnsapys1s-17kdzwq"
+modified = 2025-10-22T22:38:38.464Z
+upstream = ["CVE-2022-44940"]
+references = ["https://github.com/NixOS/patchelf/pull/419", "https://github.com/NixOS/patchelf/pull/419"]
+
+[[affected]]
+pkg = "Patchelf_jll"
+ranges = ["< 0.17.2+0", "= 2019.10.23+0"]
```

Comment:
Hm, this isn't ideal, but:

* OSV can't have both open and closed upper bounds in the same events array
* OSV also doesn't like non-existent version numbers, so we can't invent a version `2019.10.24`

```suggestion
ranges = ["<= 0.14.3+0", "= 2019.10.23+0"]
```